synapse/lib/types.py

```diff
@@ -712,9 +708,9 @@ def norm(self, valu):
             mesg = f'Value ({valu}) is too small for hugenum.'
             raise s_exc.BadTypeValu(mesg=mesg)
 
-        if self.opts.get('norm'):
-            huge.normalize(), {}
-        return huge.to_eng_string(), {}
+        huge = s_common.hugeround(huge).normalize(s_common.hugectx)
```

Comment:
So with this change, should we now raise for abs(valu)<1E-15?  i.e. is rounding 1E-20 to 0 the expected behavior?

Reply:
Seems like rounding 1E-20 to 0 would be the expected behavior. We could have a special case for abs(valu)<1E-15 but not sure if that is more or less confusing if we're making 1.0000000000000001 is round to 1